pipeline/pyproject.toml

```diff
@@ -10,6 +10,9 @@ PyPDF2 = "^1.26.0"
 pdfservices-sdk = "^1.0.1"
 navigator = {path = "../common", develop = true }
 tqdm = "^4.62.3"
+numpy = "^1.22.3"
+pandas = "^1.4.1"
+english-words = "^1.1.0"
```

Comment:
only handles english.

Reply:
I suggest you use nltk as above - you'll likely need to add [`nltk.download("wordnet")`](https://stackoverflow.com/questions/68718901/what-does-nltk-downloadwordnet-accomplish) or similar to the Dockerfile too.

Reply:
Makes sense- Adobe can only handle English so we don't need to worry about building post-processors for other languages.